logstash-core/src/test/java/org/logstash/StringInterpolationTest.java

```diff
@@ -149,6 +151,24 @@ public void TestValueIsHash() throws IOException {
         assertEquals("{\"k1\":\"v\"}", StringInterpolation.evaluate(event, path));
     }
 
+    @Test
+    public void TestTimeNow() throws IOException {
+        Event event = getTestEvent();
+        String pattern = "%{{TIME_NOW}}";
+        Timestamp before = new Timestamp();
+        Timestamp result = new Timestamp(StringInterpolation.evaluate(event, pattern));
+        Timestamp after = new Timestamp();
```

Comment:
Secondary note, that you could leave out: I know that it's hard two creation of `Instant` falls into the same nanosecond, but without a minimal sleep we can't grant that the Timestamps a monotonically increasing, they could result in same value.